### DISCUSSION TITLE
Welcome to laravel-in-app-purchases Discussions!

### DISCUSSION THREAD

**Discussion by imdhemy** [imdhemy/laravel-in-app-purchases#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HilalLko** [comment]
@imdhemy 
How to get subscriptions amount which is deducted when subscriptions renew as well updated order Id for the same?

**Reply by HilalLko** [reply]
This is required to maintain transaction logs in our database (or server)

**Reply by imdhemy** [reply]
@hilal-hipster, kindly create a separate Q/A discussion for this question.
